benchmarks/util/result_uploader.py

```diff
@@ -59,13 +59,14 @@ def upload_result(result_list, metadata):
       labels_string += ",|%s:%s|" % (key, result[key])
     new_result["labels"] = labels_string[1:]
     new_result["timestamp"] = _INITIAL_TIME
-
-    bq = big_query_utils.create_big_query()
-    row = big_query_utils.make_row(str(uuid.uuid4()), new_result)
-    if not big_query_utils.insert_rows(bq, _PROJECT_ID, _DATASET,
-                                       _TABLE + "$" + _NOW,
-                                       [row]):
-      print('Error when uploading result', new_result)
+    print(labels_string)
+# 
+#     bq = big_query_utils.create_big_query()
```

Comment:
Is this intentionally commented out?

Reply:
Sorry that I forgot to uncomment this after testing.
Done.